config/default/eventAggregator.conf.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ * @author Bout Joel, <joel@taotesting.com>
+ * @license GPLv2
+ * @package generis
+ *
+ */
+
+use oat\oatbox\event\EventAggregator;
+
+return new EventAggregator(
+    [
+        'numberOfAggregatedEvents' => 10,
```

Comment:
Are you sure that the number of aggregated events should be 10 ❔

Reply:
Nope, but that is configurable so can be changed without redeployment if needed  or benchmarking will show a need